code/datums/outfits/spec_op.dm

```diff
@@ -51,6 +51,25 @@
 	GLOB.commandos.equip(H)
 	return 1
 
+/decl/hierarchy/outfit/terrorist
+	name = "Koalicja - Terrorist"
+	uniform = /obj/item/clothing/under/terran/navy/utility/koalicja
```

Comment:
Forgot this, this will only work on the torch as this outfit is not defined anywhere else.

Reply:
(The uniform, i hate being unable to make single line comments)